developer_guides/topology2/topology2.rst

```diff
@@ -1332,6 +1332,51 @@ You can use the ``-P`` switch to convert a 2.0 configuration file to the 1.0 con
 
    alsatplg <-D args=values> -P input.conf -o output.conf
 
+Split topologies
+****************
+
+Linux kernel can load multiple topologies, a topology for a single function.
+This feature is useful when a single device is disabled in specific productuds. To achieve this, you need to split
+the topology into multiple tplg files. The split topology files should be named as follows:
+
+.. code-block:: bash
+
+        sof-<platform>-<function>-id<BE id number>.tplg
+
+Where <platform> should be the platform name like mtl, lnl, etc.
+
+Where <function> should be one of
+
+.. code-block:: bash
+
+        sdca-jack
+        sdca-<n>amp where n is the amp link numbers
+        sdca-mic
+        dmic-<n>ch where n is the channel numbers. Currently, only 2ch and 4ch are supported.
+        sdca-hdmi
```

Comment:
SNDW connected display audio or HDA connected display audio in a system where other codecs are SNDW?

Reply:
HDA connected display audio in a system where other codecs are SNDW.